src/i18n/locales/en/default.json

```diff
@@ -131,6 +131,11 @@
       "expired": "Expired",
       "copyToClip": "Invite code copied to clipboard",
       "copyToClipFailed": "Copy invite code to clipboard failed"
+    },
+    "cookieConsent": {
+      "text": "This website uses cookies to enhance the user experience. ",
```

Comment:
Do we really use cookies? We use the local storage. Or do you generalize that for the general public, since they can't understand local storage? I just noticed it that way 😄